docs/contributing.md

```diff
@@ -121,3 +121,75 @@ Prerequisites: dpkg -i http://archive.ubuntu.com/ubuntu/pool/universe/c/checksec
 Always run make sdl-check command before you submit changes.
 
 Prerequisites: pip3 install bandit
+
+## Logging Policy
+
+### Levels
+
+</br>
+
+#### `ERROR`
+Log at this level errors of every ovms operation that failed 
+</br>
+
+Example:
+```
+    "Failed to create a temporary path PATH"
+```
+
+</br>
+
+#### `WARN`
+Log at this level everything that may lead to an error/need to be fix by administrator.
+</br>
+
+Example:
+```
+    "Expected version directory to be in number format. Got:DIR_NAME"
+```
+
+</br>
+
+#### `INFO` 
+Log at this level all actions that are user-driven, or any others that changes state of application:
```

Comment:
What do you mean by user-driven? 
I agree about the "changes state of application"